indexer/src/test/java/org.pdxfinder/commands/dataloaders/UniversalLoaderTest.java

```diff
@@ -0,0 +1,31 @@
+package org.pdxfinder.commands.dataloaders;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import org.pdxfinder.commands.BaseTest;
+
+import static org.junit.Assert.*;
+
+public class UniversalLoaderTest extends BaseTest {
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @InjectMocks
+    private UniversalLoader loader;
+
+    @Test
+    public void Given_BaseFilePathContainsTrailingSlash_WhenLoading_PathIsCleaned() {
+        assertEquals("file/path", loader.stripTrailingSlash("file/path/"));
```

Comment:
since stripTrailingSlash is static, you could use UniversalLoader.stripTrailingSlash () and not have to mock -- style choice though

Reply:
Done - this now calls the static class method directly